nav/nav_stack/launch/move_base.launch

```diff
@@ -0,0 +1,14 @@
+<launch>
+
+    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
+        <rosparam file="$(find nav_stack)/config/caffeine_costmap_common_params.yaml" command="load" ns="global_costmap" /> 
+        <rosparam file="$(find nav_stack)/config/caffeine_costmap_common_params.yaml" command="load" ns="local_costmap" />
+        <rosparam file="$(find nav_stack)/config/local_costmap_params.yaml" command="load" />
+        <rosparam file="$(find nav_stack)/config/global_costmap_params.yaml" command="load" /> 
+        <rosparam file="$(find nav_stack)/config/base_local_planner_params.yaml" command="load" />
+
+        <remap from="odom" to="zed/zed_node/odom"/>
```

Comment:
I'm actually quite surprised you can navigate using visual odom only. Though I would suggest using `caffeine/odom`, as it can be considered the "ground truth" as published by Gazebo